bus/bus.go

```diff
@@ -622,16 +632,17 @@ func isErrSettingsNotFound(err error) bool {
 }
 
 // New returns a new Bus.
-func New(s Syncer, cm ChainManager, tp TransactionPool, w Wallet, hdb HostDB, cs ContractStore, os ObjectStore, ss SettingStore, gs api.GougingSettings, rs api.RedundancySettings) (http.Handler, error) {
+func New(s Syncer, cm ChainManager, tp TransactionPool, w Wallet, hdb HostDB, cs ContractStore, os ObjectStore, ss SettingStore, gs api.GougingSettings, rs api.RedundancySettings, interactionsFlushInterval time.Duration) (http.Handler, error) {
 	b := &bus{
-		s:   s,
-		cm:  cm,
-		tp:  tp,
-		w:   w,
-		hdb: hdb,
-		cs:  cs,
-		os:  os,
-		ss:  ss,
+		s:                         s,
+		cm:                        cm,
+		tp:                        tp,
+		w:                         w,
+		hdb:                       hdb,
+		cs:                        cs,
+		os:                        os,
+		ss:                        ss,
+		interactionsFlushInterval: interactionsFlushInterval,
```

Comment:
I think this can be a `const` rather than being passed to the constructor. I don't want to test things by shortening intervals anyway; there are better strategies we can use.

Reply:
cc @ChrisSchinnerl